gpgopt.py

```diff
@@ -0,0 +1,106 @@
+import gnupg
+from pathlib import Path
+from os import getlogin
```

Comment:
The parts with `getlogin()` were giving me trouble on my Linux machine for some reason. I found on
[https://docs.python.org/3/library/getpass.html#getpass.getuser](https://docs.python.org/3/library/getpass.html#getpass.getuser) that `getpass.getuser()` should do what you want but I'm not sure if it works for Windows machines.

Reply:
Thanks, got it, I will change that.